atom/browser/ui/win/atom_desktop_window_tree_host_win.cc

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2015 GitHub, Inc.
+// Use of this source code is governed by the MIT license that can be
+// found in the LICENSE file.
+
+#include <shobjidl.h>
+
+#include "atom/browser/ui/win/atom_desktop_window_tree_host_win.h"
```

Comment:
The header of current .cc file should be put on the top:
http://google-styleguide.googlecode.com/svn/trunk/cppguide.html#Names_and_Order_of_Includes